layouts/partials/head.html

```diff
@@ -16,8 +16,14 @@
   {{ $style := resources.Get "css/style.css" }}
   {{ $markdown := resources.Get "css/markdown.css" }}
   {{ $syntaxHighlight := resources.Get "css/syntax-highlight.css" }}
-  {{ $css := slice $style $markdown $syntaxHighlight | resources.Concat "style.css" | resources.Minify | resources.Fingerprint }}
-  <link rel="stylesheet" href="{{ $css.Permalink }}">
+  {{ $syntaxHighlightDark := resources.Get "css/syntax-highlight-dark.css" }}
+  {{ $css := slice $style $markdown $syntaxHighlight $syntaxHighlightDark | resources.Concat "style.css" | resources.Minify | resources.Fingerprint }}
```

Comment:
Isn't `$syntaxHighlightDark` unnecessary? In light mode, the text color will be incorrect.